scripts/mod_loader/tests/vector.lua

```diff
@@ -0,0 +1,229 @@
+local testsuite = Tests.Testsuite()
+
+function testsuite.test_vector2_xy()
+	local v = math.vec2(11,22)
+
+	Assert.TableEquals({11,22}, v, "math.vec2(x, y)")
+	
+	return true
+end
+
+function testsuite.test_vector2_v2()
```

Comment:
Some of the test names here aren't very descriptive; you might want to consider changing some of them:

current | suggested
------------ | -------------
`test_vector2_v2` | `test_vector2_copy_constructor`
`test_vector3_v2z` | `test_vector3_constructor_from_vector2`

...and so on.

These functions aren't manually called by anyone, so it's okay if they're somewhat long-winded (unless you invoke them by hand from the in-game lua console; personally I quickly got tired of that and created the testing window :) ).

Ideally, test names should be clear enough for someone without any knowledge of their implementation to know what they're testing (or know what they're supposed to test, so that they can be fixed in the event that they're broken) - this is not always easy :)

Reply:
That makes sense. I guess I got lazy when naming them to start testing and never revisited.

Reply:
Renamed tests to be more descriptive